contracts/governance_admin/src/execute/resume_reward_schedule_creation.rs

```diff
@@ -16,17 +22,24 @@ pub fn execute_resume_reward_schedule_creation(
     let mut reward_schedule_creation_request =
         REWARD_SCHEDULE_REQUESTS.load(deps.storage, reward_schedule_creation_request_id)?;
 
+    let mut event = Event::from_info(
+        concatcp!(CONTRACT_NAME, "::resume_reward_schedule_creation"),
+        &info,
+    );
+
     // mark the request as done
     match reward_schedule_creation_request.status {
         RewardSchedulesCreationRequestStatus::NonProposalRewardSchedule => {
             reward_schedule_creation_request.status =
                 RewardSchedulesCreationRequestStatus::RewardSchedulesCreated { proposal_id: None };
+            event = event.add_attribute("proposal_id", "-1".to_string());
```

Comment:
It's fine to skip a value. The absence of an attribute indicates that it is null.